### ISSUE TITLE
Stage output is not reachable via screen reader

### ISSUE THREAD

**Issue by phyllisUW** [wordplaydev/wordplay#465]
## Expected behavior

I expect that the moving Christmas tree image on the stage page has incorporated alt text or a screen reader into the code so that there is accessibility for users with visual impairments. 

## Actual behavior

After looking at the code and testing the website in different browser versions, I see that there is no existing alt text for the Christmas tree image. Users relying on screen readers for information from this page will not be able to understand the main content about this page, missing a lot of context and information.

Possible sample code: 
` <img src="🎄️" alt="A decorated Christmas tree emoji with colorful ornaments and a star on top moving across the page">`

1. Go to 'Motion'
2. Click on '..Christmas.'
3. Right Click and turn on Screen Reader

## Environment

Google Chrome Version 124.0.6367.207
Windows 10 

## Design specification

Edit the HTML and add alt text attribute that describes the image so it can be read by screen readers to users with visual impairments. Should respond and follow the assistive technologies guidelines: https://www.w3.org/WAI/WCAG22/Understanding/non-text-content#dfn-assistive-technology

**Comment by amyjko** [comment]
Thanks for reporting @phyllisUW! Updating the title here, since this issue isn't specific to the Christmas project; the issue is that the output view isn't focusable, and so the screen reader can't focus on the output content to read it. (If it could, it would say "Christmas tree").
